src/beacon/drand.rs

```diff
@@ -75,17 +71,10 @@ impl BeaconSchedule {
         parent_epoch: ChainEpoch,
         prev: &BeaconEntry,
     ) -> Result<Vec<BeaconEntry>, anyhow::Error> {
-        let (cb_epoch, curr_beacon) = self.beacon_for_epoch(epoch)?;
-        let (pb_epoch, _) = self.beacon_for_epoch(parent_epoch)?;
-        if cb_epoch != pb_epoch {
-            // Fork logic, take entries from the last two rounds of the new beacon.
-            let round = curr_beacon.max_beacon_round_for_epoch(network_version, epoch);
-            let mut entries = Vec::with_capacity(2);
-            entries.push(curr_beacon.entry(round - 1).await?);
-            entries.push(curr_beacon.entry(round).await?);
-            return Ok(entries);
-        }
+        let (_cb_epoch, curr_beacon) = self.beacon_for_epoch(epoch)?;
+        let (_pb_epoch, _) = self.beacon_for_epoch(parent_epoch)?;
         let max_round = curr_beacon.max_beacon_round_for_epoch(network_version, epoch);
+        // We don't expect this to ever be the case
```

Comment:
should we log or `panic!` on this branch?

Reply:
This comment and logic are from the [lotus PR](https://github.com/filecoin-project/lotus/pull/11572/files#diff-587eaf0df6b60dbf741d19bbe39439f6f48ffe171e82a71c76b82484ba48f386R138), I'm not 100% sure if it's true and I'd be careful adding panic here to crash forest in some edge cases

```go
        // We don't expect this to ever be the case
	if maxRound == prev.Round {
		return nil, nil
	}
```

Reply:
let's add at least a massive error log so there's a chance it catches our attention, just in case

Reply:
@LesnyRumcajs Log added, please re-approve if it looks good.